Editor/Scenes/SongEditor/SongEditor.gd

```diff
@@ -2,10 +2,13 @@ extends VBoxContainer
 
 # Signals
 signal playback_finished()
+signal song_script_error(error)
 signal track_pressed (name)
 
+onready var BIN = OS.get_executable_path()
 const BASE_SONG_SCRIPT = """extends SongScript
 
+
```

Comment:
Extra line? :thinking:

Reply:
Thats based on godot's language formatting. Even if you codebase is not formatted based on gd script formatting rules the inapp code editor must be?

Reply:
Ahh then no probs